### PR TITLE
appveyor & travis - build.sh updates

### PR DESCRIPTION
**Pull request by ttodua**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -14,16 +14,15 @@ before_install:
...
 - git fetch --depth=1 --no-tags --verbose --progress
-- pip install requests pyopenssl
-- pip install --ignore-installed six
-- pip install tox twine
-- pip install aiohttp cryptography
+- pip install tox cryptography requests # common packages
+- pip install twine # used to publish python package
+- pip install pyopenssl aiohttp # used to run real-time tests
 - php -i | grep php.ini
-- php --version
```
Comment (ttodua) [inline .travis.yml:26]: this line is needed, as above command php -i already exposes the version, i.e. see 1596th line: https://app.travis-ci.com/github/ccxt/ccxt/builds/263197729#L1595

File: .travis.yml [hunk 2]
```diff
@@ -14,16 +14,15 @@ before_install:
...
 - git checkout package.json package-lock.json
 - git config pull.rebase false
 - git fetch --depth=1 --no-tags --verbose --progress
-- pip install requests pyopenssl
-- pip install --ignore-installed six
-- pip install tox twine
-- pip install aiohttp cryptography
+- pip install tox cryptography requests # common packages
+- pip install twine # used to publish python package
+- pip install pyopenssl aiohttp # used to run real-time tests
```
Comment (ttodua) [inline .travis.yml:24]: the commands are same, just they are sorted to be similar in `appveryor.yml` , thus more readable when looking into both `.yml` files 

also, added comments aside of those lines, to explain why we have them in `travis.yml` and not in `appveyor.yml`

File: appveyor.yml [hunk 3]
```diff
@@ -8,23 +8,15 @@ install:
   - set CI=true
   # - npm install --global npm@latest
   - set PATH=%APPDATA%\npm;C:\Python38;C:\Python38\Scripts;%PATH%
-  # Remove python2.7 from path
-  - set PATH=%PATH:C:\Python27;=%
-  - set PATH=%PATH:C:\Python27\Scripts;=%
-  - echo %PATH%
```
Comment (ttodua) [inline appveyor.yml:14]: we long ago dropped support for 2.7

File: appveyor.yml [hunk 4]
```diff
@@ -8,23 +8,15 @@ install:
...
   - npm install
-  - python --version
-  - pip --version
```
Comment (ttodua) [inline appveyor.yml:17]: I've moved these commands into `test_script:` block, where are other `--version` outputs.

File: appveyor.yml [hunk 5]
```diff
@@ -8,23 +8,15 @@ install:
...
   - python -m pip install --upgrade pip setuptools
-  - pip install tox
-  - pip install requests
-  - pip install cryptography
+  - pip install tox cryptography requests # common packages
```
Comment (ttodua) [inline appveyor.yml:13]: these 3 commands are now inline (exactly in travis.yml)

**Comment by carlosmiei** [comment]
LGTM ✅

**Review by carlosmiei — APPROVED** [review]

**Pull request merged**